tokens/properties/alias.json

```diff
@@ -178,13 +178,18 @@
                 "disabled": {
                     "value": "{color.base.70.value}"
                 },
-                "field-disabled": {
-                    "value": "{color.base.80.value}"
-                },
                 "loader": {
                     "value": "{color.base.30.value}"
                 }
             },
+            "field": {
```

Comment:
Mental note: I don't think these are proper alias tokens. A general restructuring and renaming of Alias tokens is a must. To whoever is reading this: no worries, components won't be affected 😎